### PR TITLE
typo vs webedit who would win

### PR DESCRIPTION
**Pull request by deltanedas**
## About the PR
webedit, every time

**Media**
- [X] This PR does not require an ingame showcase

### CODE REVIEW

**Comment by EmoGarbage404** [comment]
The wording is still kinda weird for the last one.

File: Resources/Locale/en-US/headset/headset-component.ftl [hunk 1]
```diff
@@ -15,8 +15,8 @@ examine-headset-channels-prefix = A small screen on the headset displays the fol
 examine-headset-channel = [color={$color}]{$key} for {$id} ({$freq})[/color]
 examine-headset-no-keys = It seems broken. There are no encryption keys in it.
 examine-headset-chat-prefix = Use this {$prefix} for your department's frequency.
-examine-headset-default-channel = It's indicates that default channel of this headset is [color={$color}]{$channel}[/color].
-examine-encryption-key-default-channel = It's seems like [color={$color}]{$channel}[/color] is a default channel.
+examine-headset-default-channel = It indicates that default channel of this headset is [color={$color}]{$channel}[/color].
```
Comment (Visne) [inline Resources/Locale/en-US/headset/headset-component.ftl:18]:
```suggestion
examine-headset-default-channel = It indicates that the default channel of this headset is [color={$color}]{$channel}[/color].
```

**Comment by deltanedas** [comment]
somewhat better

**Comment by Visne** [comment]
If you could also fix the extra `'` at the end of line 888 of `posters.yml` you would be a legend

**Review by Visne — APPROVED** [review]

**Pull request merged**